config/manager/manager.template.yaml

```diff
@@ -0,0 +1,45 @@
+kind: ServiceAccount
```

Comment:
Can you remove the `manager.yaml` file and add it to `.gitignore`? It will be recreated in makefile using this template.